### PR TITLE
Add "Overlay Wedge"

### PR DESCRIPTION
**Pull request by TheRazgriz**
Script for "Overlay Wedge"

https://yugipedia.com/wiki/Overlay_Wedge

@ClementLouis 
@pyrQ 
@larry126

### CODE REVIEW

**Comment by TheRazgriz** [comment]
I should also note that the image they have on Pedia isn't great. The anime actually shows the text in a scene seconds after that picture.

File: unofficial/c511027118.lua [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	end  
+end
+function s.checkop(e,tp,eg,ep,ev,re,r,rp)
+	local cid=Duel.GetCurrentChain()
+	if cid>0 then
+		c511027118[0]=Duel.GetChainInfo(cid,CHAININFO_CHAIN_ID)
+	end
+end
+function s.condition(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.GetChainInfo(0,CHAININFO_CHAIN_ID)==c511027118[0] and Duel.IsChainDisablable(ev)
```
Comment (larry126) [inline unofficial/c511027118.lua:33]: Having
```lua
re:IsActiveType(TYPE_XYZ)
```
would be better incase Xyz Monster lost Xyz Type (looking at you , En Birds)

File: unofficial/c511027118.lua [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+end
+function s.accon(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.IsExistingMatchingCard(s.acfilter,0,LOCATION_MZONE,LOCATION_MZONE,1,nil) 
+end
+function s.aclimit(e,re)
+	local g=Duel.GetMatchingGroup(s.acfilter,0,LOCATION_MZONE,LOCATION_MZONE,nil)
+	if not re:IsActiveType(TYPE_XYZ) or g:GetMaxGroup(Card.GetRank):IsContains(re:GetHandler()) then return false end
+	local eff={re:GetHandler():GetCardEffect(511002571)}
+	for _,ree in ipairs(eff) do
+		if re==ree:GetLabelObject() then return true end
```
Comment (larry126) [inline unofficial/c511027118.lua:63]: Change the part in the loop to
```lua
local te=ree:GetLabelObject()
if te:GetType()&EFFECT_TYPE_GRANT==EFFECT_TYPE_GRANT then te=te:GetLabelObject() end
if re==te then return true end
```
This is a workaround needed to work with other cards that could access "detach ORU as cost" effect that are gained under certain conditions (e.g. RUM - Cipher Pursuit, CXyz Barian Hope Anime)

**Comment by TheRazgriz** [comment]
Updated script with recommended changes

**Pull request merged**